packages/@coorpacademy-components/src/molecule/cm-popin/index.js

```diff
@@ -190,6 +217,7 @@ const CMPopin = props => {
             ) : null}
           </div>
         ) : null}
+        {items ? <div className={style.itemsContainer}>{renderItems()}</div> : null}
```

Comment:
I think that items should be shown before the buttons (even, if it's not what we're gonna use, if someone someday will activate the buttons, we'll have this

![Capture d’écran 2023-08-10 à 11 46 14](https://github.com/CoorpAcademy/components/assets/7602475/c5ae2aa0-1078-418e-9ce3-168d3393638a)